apps/judicial-system/web/src/routes/Defender/Cases/Cases.strings.ts

```diff
@@ -22,4 +22,14 @@ export const defenderCases = defineMessages({
     defaultMessage: 'Afgreidd mál',
     description: 'Notaður sem titill á flipa sem sýnir afgreidd mál',
   },
+  noActiveCases: {
+    id: 'judicial.system.defender_cases:no_active_cases',
```

Comment:
Consider replacing ':' with '.' to be consistent with other ids.

Reply:
good eye! 🙏